src/component/handlers/edit/editOnPaste.js

```diff
@@ -176,4 +186,15 @@ function insertFragment(
   );
 }
 
+function areTextBlocksAndClipboardEqual(
+  textBlocks: Array<string>,
+  blockMap: BlockMap
+): boolean {
+  let i = 0;
+  return (
+    textBlocks.length === blockMap.size &&
+    blockMap.toSeq().every((block) => block.getText() === textBlocks[i++])
```

Comment:
`blockMap.toSeq().every` does not include the index of the iteration: https://facebook.github.io/immutable-js/docs/#/Iterable/every (easily confused with: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) 

Because of this I made a small tweak to @hellendag's proposed code.